json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -198,7 +205,7 @@ public function get_settings_response() {
 					'comment_max_links'       => (int) get_option( 'comment_max_links' ),
 					'moderation_keys'         => get_option( 'moderation_keys' ),
 					'blacklist_keys'          => get_option( 'blacklist_keys' ),
-					'lang_id'                 => get_option( 'lang_id' ),
+					'lang_id'                 => get_lang_id_by_code( wpcom_l10n_get_blog_locale_variant( $blog_id, true ) ),
```

Comment:
I think `get_lang_id_by_code()` is a wpcom specific function, and we should include a polyfill in Jetpack.

It was also used in Jetpack here: https://github.com/Automattic/jetpack/commit/8818927dc8c021d86dfad4369a52dabef7ad8b71#diff-6b016d7879a6d3fb62212af9db70b6a4R47 -- cc: @eliorivero